tests/feat/hello.spec.ts

```diff
@@ -22,8 +22,8 @@ describe('feat.Hello', () => {
     expect(headers['access-control-allow-credentials']).toEqual('true');
   });
 
-  it('should respond with "Hello"', async () => {
+  it('should respond with the correct message', async () => {
     const { data } = await makeRequest();
-    expect(data.data.response).toEqual('Hello');
+    expect(data.message).toEqual('hello nobody');
```

Comment:
This needs reverting.

Reply:
Thanks for picking this up. Oversight from me.

Reply:
I've reverted the change here and the test is now passing. I have left the change on line 25, because I think that the description generally should give more of an generalised indication of what this test is expecting, rather than stating it expects specific information back.

Reply:
I do agree (though in general I'd have to say it depends). It just irks me that the tests are included in the diff 😄